### ISSUE TITLE
Jay and Silent Bob: Mall Brawl

### ISSUE THREAD

**Issue by kjetil-f** [farmerbb/RED-Project#81]
As far as I understand it, Mall Brawl was first released on a physical NES cartridge from Limited Run, before being ported to Steam/GOG and consoles. By the looks of it the ports seems identical to the NES version.

Do anyone know if the ports are just a ROM wrapped in a NES emulator. And in that case, is it possible to extract it?

**Comment by einstein95** [comment]
Looking at the Steam version, and like "Eskimo Bob: Starring Alfonzo" from the same developer, it appears so. You'd need to extract the ROM from the Unity resource file.

Their other game, "Alfonzo's Arctic Adventure", stores the ROM file loose in the game's data folder.

**Comment by kjetil-f** [comment]
Nice. I turns out, I actually have Eskimo Bob as well, so need to test that one out.

Had a look, and could not make to much sense of it. What file is the "Unity resource file" exactly, and what method do I use to extract the ROM?

**Comment by TheNametag** [comment]
Eskimo Bob: Starring Alfonzo on Steam has "eskimobob.nes" in its root folder.  Been there the whole time, no need to dig into the unity files to collect it.  Or is there a different version of the ROM in there?

**Comment by kjetil-f** [comment]
Yeah, I know. I was actually referring to the Mall Brawl game, but the sentence came out wrong.
